src/Models/AuditActivity.php

```diff
@@ -43,20 +46,20 @@ class AuditActivity extends BaseModel
         'ip_address'
     ];
 
-    protected $dates = [
-        'created_at'
+    protected $casts = [
+        'created_at' => 'datetime'
     ];
 
     //--------------------------------------------------------------------------------------------------------------
     // Mutators
     //--------------------------------------------------------------------------------------------------------------
 
-    protected function setIpAddressAttribute($value)
+    protected function setIpAddressAttribute(string|null $value): void
     {
-        $this->attributes['ip_address'] = inet_pton($value);
+        $this->attributes['ip_address'] = inet_pton((string)$value);
```

Comment:
Isn't a null value coerced to an empty string anyway?

Reply:
```
Deprecated: inet_pton(): Passing null to parameter #1 ($ip) of type string is deprecated in /home/user/scripts/code.php on line 2
```

Sadly not :(